app/assets/javascripts/views/response_list_view.js

```diff
@@ -1,3 +1,5 @@
+var CLONE_MARKER = "CLONE_MARKER";
```

Comment:
what is a clone marker?

Reply:
I'll add a code comment -- it marks elements as having been added automatically (cloned into the form), so that we can destroy them automatically later without fear.

It's for the case when you open the modal, close it, then open it again -- we clean up first by destroying all the old clones.